### DISCUSSION TITLE
PAT Basic 1030. 完美数列 (25) (C语言实现) | PAT Solution in C

### DISCUSSION THREAD

**Discussion by xlucn** [xlucn/PAT#78] [General]
https://oliverlew.github.io/PAT/Basic/1030.html 

 题目

**Comment by Huuuuu123** [comment]
mp 可能会大于32位，我靠。。。。想了一万年

**Comment by Huuuuu123** [comment]
哭了

**Comment by ShaobinChen-AH** [comment]
请问那个while循环中1L是啥意思？

**Comment by xlucn** [comment]
> @csb-github
> 请问那个while循环中1L是啥意思？

长整型

**Comment by ShaobinChen-AH** [comment]
> @OliverLew
> > @csb-github
> > 请问那个while循环中1L是啥意思？
> 
> 长整型

还有这种写法啊，见识了，多谢！
